src/main/resources/db/migration/V1__create_table_users.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE IF NOT EXISTS "users" (
+  "uuid" UUID DEFAULT gen_random_uuid () PRIMARY KEY,
+  "first_name" varchar(20),
+  "last_name" varchar(20),
+  "username" varchar(20),
+  "email" varchar(50) NOT NULL,
+  "password" varchar(255) NOT NULL,
```

Comment:
Use the text type instead of varchar. If you want to limit the input size, do so through the application rather than the database.